src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -99,7 +131,51 @@ const ArtworkSidebarCommerialButtons: React.FC<ArtworkSidebarCommercialButtonsPr
     showInquiry({ enableCreateAlert: true })
   }
 
-  const handleCreateOrder = () => {
+  const handleCreatePartnerOfferOrder = async () => {
```

Comment:
Might be more clear if this function expects activePartnerOffer and throws error if it's called without it. If I understand correctly currently it would just silently do nothing if called without offer. Seems like it can introduce some errors where it's called in a wrong place and would be tricky to debug as UI will just do nothing.